fenix/app/src/main/java/org/mozilla/fenix/tabstray/browser/InactiveTabViewHolder.kt

```diff
@@ -75,9 +88,68 @@ class InactiveTabViewHolder(
                 onTabClick = interactor::onInactiveTabClicked,
                 onTabCloseClick = interactor::onInactiveTabClosed,
             )
+            val settings = components.settings
+            if (settings.shouldShowInactiveTabsOnboardingPopup &&
+                settings.canShowCfr
+            ) {
+                CreateInactiveTabCFR(settings)
+            }
         }
     }
 
+    /**
+     * Create the CFR needed to inform the user about inactive tabs.
+     */
+    @Composable
+    @Suppress("MagicNumber")
+    fun CreateInactiveTabCFR(settings: Settings) {
+        CFRPopup(
+            anchor = composeView,
+            properties = CFRPopupProperties(
+                popupBodyColors = listOf(
+                    FirefoxTheme.colors.gradientEnd.toArgb(),
+                    FirefoxTheme.colors.gradientStart.toArgb(),
+                ),
+                dismissButtonColor = FirefoxTheme.colors.iconOnColor.toArgb(),
+                indicatorDirection = CFRPopup.IndicatorDirection.UP,
+                popupVerticalOffset = (-25).dp, // Offset to point the arrow at the header of the list.
+            ),
+            onDismiss = {
+                settings.shouldShowInactiveTabsOnboardingPopup = false
+                org.mozilla.fenix.GleanMetrics.TabsTray.inactiveTabsCfrDismissed.record(
```

Comment:
let's use a type alias here to clean up the metric call. There's already one in this file actually. so this can instead be
```suggestion
                TabsTrayMetrics.inactiveTabsCfrDismissed.record(
```